components/brave_rewards/browser/rewards_service_impl.cc

```diff
@@ -2250,6 +2250,37 @@ void RewardsServiceImpl::SaveTwitterPublisherInfo(
                      std::move(callback)));
 }
 
+void RewardsServiceImpl::OnRedditPublisherInfoSaved(
+    SaveMediaInfoCallback callback,
+    int32_t result,
+    ledger::PublisherInfoPtr publisher) {
+  if (!Connected()) {
+    std::move(callback).Run(nullptr);
+    return;
+  }
+  if (Connected()) {
```

Comment:
Seems like we don't need the `if` here, since we check `!Connected()` above.

Reply:
Done